Client/Frontend/Brave Today/Composer/FeedDataSource.swift

```diff
@@ -74,11 +74,17 @@ class FeedDataSource {
   // MARK: - Initialization
 
   init() {
+    self.languageCode = Locale.preferredLanguages.first?.prefix(2).lowercased() ?? FeedDataSource.supportedLanguages.first!
```

Comment:
Is force unwrap safe here? Perhaps we could fallback to english locale instead

Reply:
Should be safe as `supportedLanguages` will never be empty. I just didn't want to hardcode "en" in another place :P